lib/sparse/src/common/sparse_vector_fixture.rs

```diff
@@ -2,23 +2,33 @@ use rand::Rng;
 
 use crate::common::sparse_vector::SparseVector;
 
+const MIN_WEIGHT: f64 = -100.0;
```

Comment:
As a small nitpick I can propose to define a constant range instead of min-max values